### ISSUE TITLE
[BUG] Unable to set image attributes such as width and alt text

### ISSUE THREAD

**Issue by 5uie** [org-roam/org-roam-ui#262]
**Describe the bug**
Unable to set image attributes such as width, caption and alt text

**Apologies if I have posted in the wrong section/type or if I have made a mistake with the settings!**

**To Reproduce**

1. I include an image in org as shown below, this also allows for toggling display of images inline with C-c C-x C-v,
```
  #+caption: Museum of Curiosities (Source: Ole Worm creator QS:P170,Q463570 (https://commons.wikimedia.org/wiki/File:Museum_Wormiani_Historia_1655_Wellcome_L0000128.jpg), https://creativecommons.org/licenses/by/4.0/legalcode)
  #+name: MoC
  #+attr_html:  width="500px" alt="Museum of Curiosities"
  #+attr_org:  :width 200px
   [[./images/Museum_Wormiani_Historia_1655_Wellcome_L0000128.jpg]]
```
2. Launch org-roam-ui with `M-x org-roam-ui-mode`
3. Inspect the page in both Firefox and Chrome (version details below)
4. As shown in the screenshot below, the alt-text is not setup as required and neither are the width of the image

**Expected behavior**
Alt text to be set to "Museum of Curiosities" and width set to "500px"

**Screenshots**
![orgroamui-screenshit](https://user-images.githubusercontent.com/104305869/189957761-3aef2cd8-982a-4cbb-86e1-8977937abf27.png)

**What browser were you using?**
Firefox (104.0.2 (64-bit)) & Chrome (Version 105.0.5195.102 (Official Build, ungoogled-chromium) (64-bit))

**Comment by tefkah** [comment]
i might update this soon, nextjs has a new image component that makes this a lot easier than it used to be

**Comment by 5uie** [comment]
Thank you, will wait for the update :-)
